com.zsmartsystems.zigbee/src/main/java/com/zsmartsystems/zigbee/aps/ZigBeeApsFrame.java

```diff
@@ -426,4 +426,16 @@ public String toString() {
         return builder.toString();
     }
 
+    public void oneFragmentCompleted() {
```

Comment:
Please can you add a line or two of Javadoc to this since it's a public method (I know this class isn't the best example, but I like to try and have the documentation.